File: node/src/masking.ts
/**
 * 시크릿/토큰을 로그·문자열 표현에 노출하지 않기 위한 완전 불투명 마스킹.
 * 접두 노출 없이 항상 `***`를 반환한다(값 존재 여부·길이도 노출하지 않는다).
 */
export function mask(_value?: string | null): string {
  return '***'
}

File: ruby/lib/keycloak_sdk/masking.rb
# frozen_string_literal: true

module KeycloakSdk
  # 완전 불투명 마스킹(접두 노출 없음).
  module Masking
    module_function

    def mask(secret)
      secret.nil? ? nil : "***"
    end
  end
end
